wrapper/src/test/java/integration/refactored/container/tests/ReadWriteSplittingTests.java

```diff
@@ -600,11 +592,14 @@ public void test_pooledConnection_reuseCachedConnection() throws SQLException {
     }
   }
 
-  protected static HikariConfig getHikariConfig(HostSpec hostSpec, Properties props) {
-    final HikariConfig config = new HikariConfig();
-    config.setMaximumPoolSize(1);
-    config.setInitializationFailTimeout(20000);
-    return config;
+  protected HikariPoolConfigurator getHikariConfig(int maxPoolSize) {
+    return (hostSpec, props) -> {
+      final HikariConfig config = new HikariConfig();
+      config.setMaximumPoolSize(maxPoolSize);
+      config.setInitializationFailTimeout(75000);
+      config.setKeepaliveTime(30000);
```

Comment:
TCPKeepAlive feature doesn't work well with ToxyProxy. Is it essential setting for the tests?